react_app/src/reducers/index.js

```diff
@@ -55,7 +54,9 @@ const allReducers = combineReducers({
     toggleAddReviewBox: addReviewBox,
     addReviewRating: addReviewRating,
     addReviewReview: addReviewReview,
-    reviewList: reviewReducer
+    reviewList: reviewReducer,
+    countPage: countPage, 
```

Comment:
determines how many pages there are. In other words, how many pages are needed to display all the courses.